app/controllers/idv/in_person/verify_info_controller.rb

```diff
@@ -152,13 +143,13 @@ def analytics_arguments
       def process_async_state(current_async_state)
         if current_async_state.none?
           idv_session.resolution_successful = false
-          render 'idv/verify_info/show'
+          render 'idv/in_person/verify_info/show'
         elsif current_async_state.in_progress?
           render 'shared/wait'
         elsif current_async_state.missing?
           analytics.idv_proofing_resolution_result_missing
           flash.now[:error] = I18n.t('idv.failure.timeout')
-          render 'idv/verify_info/show'
+          render 'idv/in_person/verify_info/show'
```

Comment:
I had to change this because I was using the shared view, but you might be able to use `render :show` here and on line 146, and then you might be able to use the shared `process_async_state` in VerifyInfoConcern if they haven't diverged anywhere else.

Reply:
ohh, great catch, thank you. I moved the method into the concern in 7ddff6d39955caf1d59a7ed196c0916e222955d7

Reply:
Looks good! Tried out the flow and everything works.